wifinina: panic when trying to connect to AP without password

When trying to use the WiFiNINA driver, if you attempt to connect to an unsecured access point, you get the following error:

```
panic: runtime error: index out of range
```

It appears this is because the firmware expects an empty string param to have length of zero, and then not actually send any data for the null string. I will submit a PR with a fix that appears to address this shortly.

--- a/wifinina/wifinina.go
+++ b/wifinina/wifinina.go
@@ -969,7 +969,9 @@ func (d *Device) sendParamBuf(p []byte, isLastParam bool) (l int) {
 func (d *Device) sendParamStr(p string, isLastParam bool) (l int) {
 	l = len(p)
 	d.SPI.Transfer(uint8(l))
-	d.SPI.Tx([]byte(p), nil)
+	if l > 0 {
+		d.SPI.Tx([]byte(p), nil)
+	}
 	if isLastParam {
 		d.SPI.Transfer(CmdEnd)
 		l += 1

